### DISCUSSION TITLE
Error Category

### DISCUSSION THREAD

**Discussion by AndyGauge** [rust-lang/crates.io#4334] [Q&A]
When I search for Error on Crates.io I get 1072 results, and most of them have to deal with handling errors in Rust.  When I search categories for error, I get nothing.  I find error-chain categorized as rust patterns.  Failure is not categorized.  Is Rust Patterns the best place for error handling or should it be a separate category?

**Comment by kornelski** [comment]
Crates tagged error: https://crates.io/keywords/error

Most of them are related to Rust errors, so it could be a sub-category of Rust Patterns category. But there's an exception: https://crates.io/crates/win32-error
